src/main/drivers/bus_i2c.h

```diff
@@ -64,9 +80,20 @@ typedef struct i2cState_s {
     volatile uint8_t* read_p;
 } i2cState_t;
 
+typedef struct i2cPinConfig_s {
+    ioTag_t ioTagSCL[I2CDEV_MAX];
```

Comment:
Personally I would prefer to have `i2cPinConfig` as array of SCL+SDA pairs - parameter groups will work better.
 But current CLI resource implementation may complicate that ..